### DISCUSSION TITLE
网站报错了，另外大大可以开源你网站那个排盘的那个网站吗

### DISCUSSION THREAD

**Discussion by 104260033** [SylarLong/iztro#193] [General]
main-dafdb669.js:626 Uncaught TypeError: Cannot read properties of undefined (reading 'toLocaleLowerCase')
    at r (main-dafdb669.js:626:15729)

**Comment by SylarLong** [comment]
麻烦留下出错的链接

**Reply by 104260033** [reply]
<img width="1339" alt="image" src="https://github.com/SylarLong/iztro/assets/5122595/6e785e15-c1d9-42d4-8ef3-99b0c9df1ced">
https://ziwei.pub/astrolabe/?d=1961-3-5&t=1&leap=false&g=female&type=solar&n=

**Comment by SylarLong** [comment]
@104260033 感谢大佬的报告，bug已修复。enjoy～

**Comment by 104260033** [comment]
你才是真正的大佬

**Comment by 104260033** [comment]
这里有个疑惑，我找到一些资料里面排盘的结果与这里的不一致，不一致的点在于，资料上显示庙旺，但是这边是陷地

**Reply by SylarLong** [reply]
能提供一下资料么？

**Reply by 104260033** [reply]
<img width="1449" alt="image" src="https://github.com/SylarLong/iztro/assets/5122595/068590ed-4f72-42ab-b599-7c9464c7e9b8">
大概是这样的，特别是这个化忌很重要

**Reply by SylarLong** [reply]
我所使用的安星诀来自于《紫微斗数全书》，对于昌曲的安法如下：

>辰上顺时文曲位，戌上逆时觅文昌

详见 [安辅弼、昌曲、空劫诀](https://ziwei.pro/learn/setup.html#_13-%E5%AE%89%E8%BE%85%E5%BC%BC%E3%80%81%E6%98%8C%E6%9B%B2%E3%80%81%E7%A9%BA%E5%8A%AB%E8%AF%80)

不知道老师的安星诀是根据哪一个版本，能否提供一下资料？另外，关于星耀的亮度，也希望能有相关资料。如果不愿意提供资料，可以在开发过程中全局配置自己门派的亮度，详见 [开发文档](https://ziwei.pro/posts/config-n-plugin.html#%E9%85%8D%E7%BD%AE)
